src/commcare_cloud/commands/terraform/templates/terraform.tf.j2

```diff
@@ -5,7 +5,8 @@ terraform {
     region  = {{ state_bucket_region|tojson }}
     encrypt = true
   }
-  required_version = ">= 0.12.0, <= 0.13.7"
+  {#- guaranteed to be a valid string; see comment in terraform_version_range_string definition #}
+  required_version = "{{ terraform_version_range_string }}"
 }
 
 provider "aws" {
```

Comment:
@dannyroberts aws provider 4.8.0 is particular for terraform13 version it will not support for terraform12. T12 supports >3.75.1. 
So we need to make a small change here, Instead of using ~> 4.8.0 we have to use ~> 3.75.1

Reply:
![image](https://user-images.githubusercontent.com/11612356/161761670-c037869a-c65a-417c-a123-6ad8cd2588ec.png)

Reply:
Does `~> 3.75.1` work for both terraform 0.12 and 0.13?

Reply:
https://github.com/dimagi/commcare-cloud/pull/5241